### PR TITLE
Quality redesign 📈

### PR DESCRIPTION
**Pull request by dullus**

### CODE REVIEW

File: src/components/quality/Quality.tsx [hunk 1]
```diff
@@ -15,34 +17,58 @@ export class Quality extends React.Component<IProps> {
...
     if (len < 2) {
-      return 'down';
+      return { color: E.LABEL_COLOR.green, direction: E.DIRECTION.steady };
     } else {
-      const lastSprint = sprints[sprints.length - 1];
-      const secondLastSprint = sprints[sprints.length - 2];
-      return secondLastSprint.quality.trend >= lastSprint.quality.trend ? 'down' : 'up';
+      const lastSprint = sprints[len - 1];
+      const secondLastSprint = sprints[len - 2];
+      return secondLastSprint.quality.trend >= lastSprint.quality.trend
```
Comment (lukasorgovan) [inline src/components/quality/Quality.tsx:56]: for the end calculation use `parseTrendStandard` from utils please -- see example of usage in `FocusFactor.tsx`

File: src/components/quality/Quality.tsx [hunk 2]
```diff
@@ -15,34 +17,58 @@ export class Quality extends React.Component<IProps> {
...
-              </p>
-            </TitleContentBox>
-          </GraphWrapper>
-          <QualityTrend direction={trend} />
-        </div>
+        <GraphWrapper>
+          <TitleContentBox title={this.getTitle()} type={E.CONTENTBOX_TYPE.graph}>
+            <QualityProgress sprints={this.props.sprints} />
+          </TitleContentBox>
+          <QualityTrend average={average} color={color} direction={direction} />
```
Comment (lukasorgovan) [inline src/components/quality/Quality.tsx:27]: use standard Trend component, please

File: src/components/qualityTrend/QualityTrend.stories.tsx [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import { DIRECTION, LABEL_COLOR } from '../../lib/enums';
```
Comment (lukasorgovan) [inline src/components/qualityTrend/QualityTrend.stories.tsx:2]: remove this file, please

File: src/components/qualityTrend/QualityTrend.tsx [hunk 4]
```diff
@@ -1,18 +1,25 @@
 import React from 'react';
-import { ReactComponent as SvgArrowTrend } from '../../icons/arrow-trend.svg';
-import style from './QualityTrend.module.css';
+import * as E from '../../lib/enums';
```
Comment (lukasorgovan) [inline src/components/qualityTrend/QualityTrend.tsx:2]: remove file use standard trend component

File: src/components/quality/Quality.tsx (hunk 1 above)
Comment (dullus) [reply src/components/quality/Quality.tsx]: Done. But careful, as here is reversed logic. Updated utils.

File: src/components/quality/Quality.tsx (hunk 2 above)
Comment (dullus) [reply src/components/quality/Quality.tsx]: Done

File: src/components/qualityTrend/QualityTrend.stories.tsx (hunk 3 above)
Comment (dullus) [reply src/components/qualityTrend/QualityTrend.stories.tsx]: Done

File: src/components/qualityTrend/QualityTrend.tsx (hunk 4 above)
Comment (dullus) [reply src/components/qualityTrend/QualityTrend.tsx]: Done

**Pull request merged**